DiscountedPrice.py

```diff
@@ -0,0 +1,17 @@
+price = input("Enter the price: ")
+
+print("The initial price is: " + str(price))
```

Comment:
You shouldn't convert price to string; it's a string when enter it from the keyboard.

Reply:
Done